templates/solutions/index.html

```diff
@@ -155,15 +157,14 @@ <h3 class="p-text--small-caps">Available on</h3>
                       }}
                     </div>
                     <div class="p-logo-section__item">
-                      {{ image(url="https://assets.ubuntu.com/v1/6418af27-ibm-logo.png",
+                        {{ image(url="https://assets.ubuntu.com/v1/43adb0a3-ibm%20cloud%20logo.png",
```

Comment:
![image](https://github.com/user-attachments/assets/d5ef32e1-976e-4a9f-aa0e-6d52039e81af)

This is broken, but I think it's cause this is the wrong version of the logo. Logo cloud assets are all supposed to be similarly sized and have top & bottom white space baked in eg, https://assets.ubuntu.com/v1/580794df-microsoft-azure-logo.png

Reply:
Thanks both, looking good now!
<img width="975" alt="Screenshot 2025-05-08 at 11 44 20" src="https://github.com/user-attachments/assets/7e893414-754e-4935-80c6-e1dedf5b4190" />